.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Test A
+
+# Checklist:
+
+- [ ] Followed the style guidelines of this project
+- [ ] Changelogs have been updated
+- [ ] Unit tests have been added
```

Comment:
Would be nice to have "Documentation has been updated, if needed"

Reply:
Done!